WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -49,13 +49,7 @@ extension LoginEpilogueUserInfo {
     /// Updates the Epilogue properties, given a SocialService instance.
     ///
     mutating func update(with service: SocialService) {
-        switch service {
-        case .google(let user):
-            fullName = user.profile?.name ?? String()
-            email = user.profile?.email ?? String()
-        case .apple(let user):
-            fullName = user.fullName
-            email = user.email
-        }
+        fullName = service.user.fullName
+        email = service.user.email
```

Comment:
At the WordPressAuthenticator level, both cases now have the same associated type (because we no longer rely on Google's `GIDGoogleUser`. `user` is a computed var on `SocialService` that hides away that `switch` and unwrapping logic. See https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/764